wust_binocular:
  ros__parameters:
    common:
      fps: 60
      max_time_diff_us: 10000
      max_infer_threads: 2
      max_delay: 1.0
    cameraR:
      target_sn: DA1094119
      acquisition_frame_rate: 200
      adc_bit_depth: Bits_8
      camera_info_path: /home/hy/wust_vision/config/camera_info.yaml
      exposure_time: 1000
      gain: 16.9
      gamma: 0.7 #Bayer用不了
      pixel_format: BayerRG8 #RGB8Packed
      acquisition_frame_rate_enable: true
      reverse_x: false
      reverse_y: false
      recorder: true
      use_high_priority: true
      high_priority_cpu_id: 3
      high_priority_cpu_priority: 90
      use_sched_fifo: true
      video_player:
        fps: 200
        use_high_priority: true
        high_priority_cpu_id: 3
        high_priority_cpu_priority: 90
        use_sched_fifo: true
        loop: true
        #path: /home/hy/wust_data/video/222.MP4
        #path: /home/hy/wust_data/video/20250726_233351.avi
        path: /home/hy/data/video_save/成都.avi
        #path: /home/hy/wust_data/video/aaa.mp4
        #path: /home/hy/wust_data/video/rune.mp4
        #path: /home/hy/wust_data/video/Sentry.mp4
        start_frame: 0
        alpha: 1.0
        beta: 0
        use: true
    cameraL:
      target_sn: DA1094119
      acquisition_frame_rate: 200
      adc_bit_depth: Bits_8
      camera_info_path: /home/hy/wust_vision/config/camera_info.yaml
      exposure_time: 1000
      gain: 16.9
      gamma: 0.7 #Bayer用不了
      pixel_format: BayerRG8 #RGB8Packed
      acquisition_frame_rate_enable: true
      reverse_x: false
      reverse_y: false
      recorder: true
      use_high_priority: true
      high_priority_cpu_id: 3
      high_priority_cpu_priority: 90
      use_sched_fifo: true
      video_player:
        fps: 200
        use_high_priority: true
        high_priority_cpu_id: 3
        high_priority_cpu_priority: 90
        use_sched_fifo: true
        loop: true
        #path: /home/hy/wust_data/video/222.MP4
        #path: /home/hy/wust_data/video/20250726_233351.avi
        path: /home/hy/data/video_save/成都.avi
        #path: /home/hy/wust_data/video/aaa.mp4
        #path: /home/hy/wust_data/video/rune.mp4
        #path: /home/hy/wust_data/video/Sentry.mp4
        start_frame: 0
        alpha: 1.0
        beta: 0
        use: true
    logger:
      log_level: INFO
      log_path: /home/hy/wust_log
      use_logcli: true
      use_logfile: true
      use_simplelog: true
